### DISCUSSION TITLE
Welcome to Wealthfolio Discussions!

### DISCUSSION THREAD

**Discussion by afadil** [afadil/wealthfolio#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nic-brownie** [comment]
Hey Afadil, I have been playing with your app and so far it has been useful. Thank you for making it available. 

I run a startup and I am experimenting with it to manage and show investors the performance of their investment.

**I have thought of the following things I would add if I could:**

- Custom symbols/tickers
- Custom activity type
- Notes/Memos

**Comment by afadil** [comment]
Hey everyone!

I just set up a Discord server for Wealthfolio community so we can share, and collaborate together
I hope you'll join us.

https://discord.gg/WDMCY6aPWK

**Reply by merryHunter** [reply]
@afadil the invite is invalid here...

**Reply by afadil** [reply]
Link updated, thank you.
https://discord.gg/WDMCY6aPWK
